### PR TITLE
CPAL.Color.fromRGBA

### PR DESCRIPTION
**Pull request by jansindl3r**
there wasn't option to initialize Color using RGBA, but instead BGRA or HEX. User inputs are mostly RGBA, before I had to use uncomfortably and unreadably `Color(*color[:3][::-1], color[3])` or switch it aside.

### CODE REVIEW

**Comment by anthrotype** [comment]
i am ok to add a classmethod `fromRGBA`, but you don't need to remove the namedtuple base class. Please put that back and remove the custom `__init__`

**Comment by jansindl3r** [comment]
done

File: Lib/fontTools/ttLib/tables/C_P_A_L_.py [hunk 1]
```diff
@@ -236,12 +236,12 @@ def fromXML(self, name, attrs, content, ttFont):
...
 
-	def hex(self):
-		return "#%02X%02X%02X%02X" % (self.red, self.green, self.blue, self.alpha)
-
 	def __repr__(self):
 		return self.hex()
 
+	def hex(self):
+		return "#%02X%02X%02X%02X" % (self.red, self.green, self.blue, self.alpha)
+
```
Comment (anthrotype) [inline Lib/fontTools/ttLib/tables/C_P_A_L_.py]: please undo this change (moving `hex` method) as it is just diff noise, thanks

Comment (jansindl3r) [reply Lib/fontTools/ttLib/tables/C_P_A_L_.py]: ye, I am learning this culture of contributing. I saw it as very small improvement in legibility

Comment (justvanrossum) [reply Lib/fontTools/ttLib/tables/C_P_A_L_.py]: It changed the object type to no longer be a tuple subclass, so it would not be backwards compatible.

Comment (anthrotype) [reply Lib/fontTools/ttLib/tables/C_P_A_L_.py]: that change was undone, it's still a namedtuple

Comment (justvanrossum) [reply Lib/fontTools/ttLib/tables/C_P_A_L_.py]: I know, I was just in an explainy mood...

Comment (anthrotype) [reply Lib/fontTools/ttLib/tables/C_P_A_L_.py]: oh ok, thank you! :)
